force-app/main/default/lwc/recurringDonationTable/recurringDonationTable.js-meta.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>53.0</apiVersion>
+    <isExposed>true</isExposed>
+    <masterLabel>Recurring Donations Table</masterLabel>
```

Comment:
@maurimart According to the ticket AC the component name should be "Recurring Donations".

Reply:
Done!